portal/messaging/messages.php

```diff
@@ -51,7 +53,8 @@
 use OpenEMR\Core\Header;
 use OpenEMR\Events\Messaging\SendSmsEvent;
 
-if (!(isset($GLOBALS['portal_onsite_two_enable'])) || !($GLOBALS['portal_onsite_two_enable'])) {
+$portal_onsite_two_enable = $globalsBag->get('portal_onsite_two_enable');
```

Comment:
GlobalsBag is a subclass of symfony parameterbag, which has a `getBoolean`. We should use that when we know a thing is a boolean.

Reply:
Fixed in all places